changelog/unreleased/pull-2195

```diff
@@ -14,4 +14,8 @@ file can be written to the file before any of the preceeding file blobs.
 It is therefore possible to have gaps in the data written to the target
 files if restore fails or interrupted by the user.
 
+The implementation will try to preallocate files on the filesysten, but
+will silently skip this step if it is not supported by the filesystem.
```

Comment:
This is user-facing documentation, please explain what benefits "preallocate" provides, i.e. much better read performance for large files like VM images due to less filesystem fragmentation.

Reply:
I've described it as "good read performance" as I didn't come up with a good reference point to compare against. And essentially the preallocation is rather about trying to avoid ruining the read performance instead of improving it.